doc/src/challenges/0014-convert-num.md

```diff
@@ -0,0 +1,138 @@
+# Challenge 14: Safety of Primitive Conversions
+
+- **Status:** Open
+- **Tracking Issue:** *None*
```

Comment:
Knowing that this is a chicken-and-egg problem: could you please create the tracking issue right away (see https://github.com/model-checking/verify-rust-std/issues/19 or several others for an example)?

Reply:
Added #220